engine/language-server/src/test/scala/org/enso/languageserver/websocket/json/ContextRegistryTest.scala

```diff
@@ -877,6 +880,63 @@ class ContextRegistryTest extends BaseServerTest {
       client.expectJson(json.executionContextVisualisationNotFound(2))
     }
 
+    "get component groups" in {
+      val client = getInitialisedWsClient()
+
+      // create context
+      client.send(json.executionContextCreateRequest(1))
+      val (requestId1, contextId) =
+        runtimeConnectorProbe.receiveN(1).head match {
+          case Api.Request(requestId, Api.CreateContextRequest(contextId)) =>
+            (requestId, contextId)
+          case msg =>
+            fail(s"Unexpected message: $msg")
+        }
+      runtimeConnectorProbe.lastSender ! Api.Response(
+        requestId1,
+        Api.CreateContextResponse(contextId)
+      )
+      client.expectJson(json.executionContextCreateResponse(1, contextId))
+
+      // get component groups
+      client.send(json.executionContextGetComponentGroupsRequest(2, contextId))
+      val requestId2 =
+        runtimeConnectorProbe.receiveN(1).head match {
+          case Api.Request(requestId, Api.GetComponentGroupsRequest()) =>
+            requestId
+          case msg =>
+            fail(s"Unexpected message: $msg")
+        }
+
+      runtimeConnectorProbe.lastSender ! Api.Response(
+        requestId2,
+        Api.GetComponentGroupsResponse(
+          TestComponentGroups.standardBase.toVector
+        )
+      )
+      client.expectJson(json"""
+          { "jsonrpc": "2.0",
+            "id": 2,
+            "result": {
+              "componentGroups": [
+                {
+                  "library" : "Standard.Base",
+                  "module" : "Input",
+                  "exports" : [
+                    {
+                      "name" : "Standard.Base.File.new"
+                    },
+                    {
+                      "name" : "Standard.Database.Connection.Database.connect"
+                    }
```

Comment:
This will likely grow. Maybe worth just checking if these two are contained in the response, but allowing other ones to be added too?
Otherwise each addition will need to adapt the tests.

Reply:
The contents of standard libraries is mocked here, and should not be updated